plugins/heroku/src/getHerokuStagingApp.ts

```diff
@@ -17,10 +18,10 @@ async function getHerokuStagingApp(): Promise<string> {
   const { name: appName }: HerokuApiResGetStaging = await heroku.get(`/apps/${appId}`)
 
   writeState('staging', { appName })
-  console.log(`retrieving details for ${appName}'s latest release...`)
+  logger.verbose(`retrieving details for ${appName}'s latest release...`)
```

Comment:
**suggestion:** perhaps we could change these to logger.info - it would be helpful to see these logged out in circleci, and convenient to not have to set the logger level to verbose to rerun the ci again to debug

Reply:
I was thinking that we should set the CI to print verbose logs by default?